lib/fixedmap/Cargo.toml

```diff
@@ -2,7 +2,3 @@
 name = "fixedmap"
 version = "0.1.0"
 edition = "2021"
-
-[lib]
-test = false
-bench = false
```

Comment:
This got removed?

Reply:
Yup, I removed the stanza from things that can compile on the host – we may want to add tests to `fixedmap` in the future!